maintainers/maintainer-list.nix

```diff
@@ -9220,6 +9220,15 @@
     githubId = 1621930;
     name = "Kamil Chmielewski";
   };
+  kamillaova = {
```

Comment:
This should be added in a separate commit and be added in this order:
    maintainers: add kamillaova
    mstpd: 0.0.8 -> 0.1.0

Reply:
Done